README.md

```diff
@@ -76,8 +76,62 @@ Then add the plugin to your Expo app config (`app.json`, `app.config.json` or `a
 }
 ```
 
+And lastly, prebuild the application so the indy-sdk can be added as native dependency:
```

Comment:
Maybe we can add: "If you aren't making any manual modification to the iOS and Android directories you can add them to the gitignore of your project and generate them on demand"